csrc/scheduler/reduction_heuristic.h

```diff
@@ -226,7 +230,11 @@ class ReductionParams : public HeuristicParams {
     if (batches_per_block_inner_reduction > 1 || persistent_kernel) {
       ss << "Batches per block: " << batches_per_block_inner_reduction << "\n";
     }
-
+    if (circular_buffer_options.isEnable()) {
```

Comment:
nitpick, this felt a bit strange to me.

CircularBufferOptions seems to have a `isEnabled()` field. I think we should explicitly print out `not enabled` in https://github.com/NVIDIA/Fuser/blob/d8b8cf4366e89ad69caa435d9c3edd9ba3e4b420/csrc/ir/interface_nodes.h#L334-L336, instead of guarding the print out in options here.

Reply:
I'll leave that change to a different PR since it also influence matmul